# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../atop/linux-2.6.23.diff
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- atoppatch-kernel-2.6.21/01patch-2.6.21_atopcnt	2007-06-07 09:56:21.000000000 +0200
+++ atoppatch-kernel-2.6.21-fixed/01patch-2.6.21_atopcnt	2007-10-15 16:07:50.000000000 +0200
@@ -22,7 +22,7 @@
 diff -u --recursive linux-2.6.20.4/fs/proc/array.c linux/fs/proc/array.c
 --- linux-2.6.20.4/fs/proc/array.c	2007-03-23 20:52:51.000000000 +0100
 +++ linux/fs/proc/array.c	2007-03-27 08:05:22.000000000 +0200
-@@ -458,6 +458,25 @@
+@@ -517,6 +517,25 @@
  		task->rt_priority,
  		task->policy,
  		(unsigned long long)delayacct_blkio_ticks(task));
@@ -45,7 +45,7 @@
 +			task->stat.raw_snd,                  /* ATOP */
 +			task->stat.raw_rcv);                 /* ATOP */
 +
- 	if(mm)
+ 	if (mm)
  		mmput(mm);
  	return res;
 diff -u --recursive linux-2.6.20.4/include/linux/sched.h linux/include/linux/sched.h
